crates/fuel-core/src/schema/tx/input.rs

```diff
@@ -298,6 +298,7 @@ impl From<&fuel_tx::Input> for Input {
                 predicate: HexString(predicate.clone()),
                 predicate_data: HexString(predicate_data.clone()),
             }),
+            input => todo!("No mapping for input {input:?}"),
```

Comment:
We might need to use a `TryFrom` instead of a `From` so we don't have panicking code.